libs/reports-list/src/lib/reports-list/reports-list.component.ts

```diff
@@ -10,7 +15,20 @@ export class ReportsListComponent implements OnInit {
   @Input()
   reportsList: VaccinesReport[];
 
-  constructor() {}
+  constructor(public dialog: MatDialog) {}
 
   ngOnInit(): void {}
+
+  openDialog(report: VaccinesReport): void {
+    const dialogRef = this.dialog.open(ReportModalComponent, {
```

Comment:
Please verify that it works with various resolution. Also, play with Windows scaling in the monitor settings window.

Reply:
@erezefrat 
Not that in the `scss` file of this component I've used `media query` to allow for small screens differentiation.
You can do the same to split to different definitions per screen if needed.

Reply:
tried changing resolutions and scaling and it looks good